fuzzers/aflcc_no_orig_dic/aflcc_mock.c

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
```

Comment:
I think dictionary name is missing at t at end
aflcc_no_orig_dict
can you move the files.

Reply:
god catch. Done.